src/buildercore/bootstrap.py

```diff
@@ -252,12 +252,17 @@ def _update_ec2_node():
         install_master_flag = str(is_master).lower() # ll: 'true' 
         master_ip = master(region, 'private_ip_address')
 
+        # TODO: this is a little gnarly. I think I'd prefer this logic in the script:
+        #       if [ cat /etc/build-vars.json | grep 'nodename' ]; then ... fi
```

Comment:
I am wondering how this works in Vagrant, do we have a fallback? I think we should handle the case of the build vars being missing at all